atomics/T1078.004/T1078.004.yaml

```diff
@@ -62,14 +62,6 @@ atomic_tests:
   supported_platforms:
   - iaas:azure
   input_arguments:
```

Comment:
username and password fields are obsolete as Azure logins all require MFA, which must be done manually via the browser. The user can enter these values themselves rather than inputting their credentials as input variables for the script to execute.

Reply:
thank you for this update!

Reply:
Np :) thank you for the quick reviews and the attention you give to my PR